tests/testthat/setup-workflow-ref.R

```diff
@@ -194,7 +194,25 @@ create_rlg_models <- function() {
   )
 }
 
-cleanup <- function() {
+clean_test_enviroment <- function(.f, env = parent.frame())
+{
+  if(missing(.f))
```

Comment:
I think the cleanup call is made before to be a bit defensive.

Reply:
> I think the cleanup call is made before to be a bit defensive.

I think so too, tough in my opinion it's good to be defensive against things like, say, arbitrary input from the user, and not something bbr's own test suite is in full control over.

Reply:
I think you are correct that further revision to the test suite should be in another PR. Since the improvements are minor, I would like to keep as an incremental improvement.